lib/orderbook/OrderBook.ts

```diff
@@ -335,9 +335,10 @@ class OrderBook extends EventEmitter {
             this.logger.info(`match executed on taker ${taker.id} and maker ${maker.id} for ${maker.quantity} with peer ${maker.peerPubKey}`);
           }
           result.swapResults.push(swapResult);
-          onUpdate && onUpdate({ case: PlaceOrderEventCase.SwapResult, payload: swapResult });
+          onUpdate && onUpdate({ type: PlaceOrderEventType.SwapResult, payload: swapResult });
         } catch (err) {
           failedSwapQuantity += portion.quantity;
+          onUpdate && onUpdate({ type: PlaceOrderEventType.FailedSwap, payload: maker });
```

Comment:
isn't there any valuable information in the error that we can pass in regards to the failure?